.github/workflows/nightly-ci.yml

```
@@ -16,6 +16,22 @@ env:
   BUILDER_NAME: b-${{ github.run_id }}-${{ github.run_attempt }}
 
 jobs:
+  # ============================================================================
+  # GATING JOBS
+  # ============================================================================
+  # This is an anchor job list used for the needs field of downstream ancillary jobs
+  _gating_jobs: &gating_jobs
```

actionlint (1.7.12)

[error] "runs-on" section is missing in job "_gating_jobs"

(syntax-check)